src/main/java/fm/last/moji/impl/FileUploadOutputStream.java

```diff
@@ -93,6 +93,7 @@ public void close() throws IOException {
             String message = httpConnection.getResponseMessage();
             int code = httpConnection.getResponseCode();
             if (HttpURLConnection.HTTP_OK != code && HttpURLConnection.HTTP_CREATED != code) {
+              log.error("Invalid response: " + httpConnection.getResponseCode() + " - " + httpConnection.getResponseMessage());
               throw new IOException(code + " " + message);
```

Comment:
Both the code and the message are in the IOException that's thrown here so there's no point logging it as it will be output wherever the exception ends up.